src/tools/rz-diff/binary_diffing.md

```diff
@@ -1,75 +1,181 @@
 # Binary Diffing
 
-This section is based on the http://rizin.today article "[binary diffing](https://rizinorg.github.io/blog/posts/binary-diffing/)"
 
-Without any parameters, `rz-diff` by default shows what bytes are changed and their corresponding offsets:
+## Distance
+For bulk processing, you may want to have a higher-level overview of differences.
+`d` option serves to calculate the distance between the two binaries using either myers algorithm or the levenstine algorithm.
+
+```
+ -d --------> myers (myers algorithm)
+       |----> leven (levenstein algorithm)
+```
+
+### Myers algorithm:
+
+`rz-diff -d myers /bin/true /bin/false`
+
+output:
+```
+similarity: 0.974
+distance: 2046
+```
+### Levenshtein distance:
+`rz-diff -d leven /bin/true /bin/false`
```

Comment:
I also recommend to add a one-two sentences description of algorithm with a link to either Wikipedia or some published paper.